src/common/kmp-json-file.ts

```diff
@@ -37,6 +37,11 @@ export interface RelevantKmpOptions {
    * The model version. The first released version **MUST** be 1.0.0.
    */
   version?: string;
+  /**
+   * The model Id.
+   * keyman will create a model Id for every models, derived from "organization, name of language, modelname"
```

Comment:
This can be updated in a new PR, but this you might want to refer to this documentation here: https://help.keyman.com/developer/current-version/guides/lexical-models/tutorial/step-2#toc--the-model-id-